src/httprpc.cpp

```diff
@@ -152,6 +152,124 @@ static bool RPCAuthorized(const std::string &strAuth,
 
 static bool HTTPReq_JSONRPC(Config &config, HTTPRequest *req,
                             const std::string &) {
+    // First, check and/or set CORS headers according to
+    // https://www.w3.org/TR/cors/#resource-requests
+    do {
```

Comment:
What is this loop for?

Reply:
Using `do { ... } while (false);` to enable the use of `break`.